app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ActiveRecord::Base
+  
+  has_many :comments, as: :commentable 
+  
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+  
+  validates :content, presence: true
+
+  default_scope -> { order(created_at: :desc) }
```

Comment:
this one is probably fine, but I would advise against ever using `default_scope`. It can really mess with you if you ever try to implement ordering of your records elsewhere, and it's often not too bad to instead define a scope that handles the ordering you want and call the scope wherever you desire this ordering.